pom.xml

```diff
@@ -322,6 +322,13 @@
                         </systemProperties>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <configuration>
+                        <argLine>-Duser.language=en -Duser.region=US</argLine>
```

Comment:
It would be better to use either `systemPropertyVariables` or `environmentVariables` (if the first one doesn't work).
Also I'm not sure what wil happen